pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -191,7 +193,21 @@ func newEndpoint(address string, port uint32) (*endpoint.LbEndpoint, error) {
 // the endpoints for the cluster.
 func updateCluster(clusterName string, edsCluster *EdsCluster) {
 	// TODO: should we lock this as well ? Once we move to event-based it may not matter.
-	hostname, ports, labels := model.ParseServiceKey(clusterName)
+	var hostname string
```

Comment:
Just for 1 month... You can add a 'deprecated' and 'TODO: delete first day after 0.8'

Reply:
lol..